polylogue/document_store.py

```
@@ -256,79 +289,87 @@ def persist_document(
+                        "updatedAt": updated_at,
+                        "createdAt": created_at,
+                        "html": html,
+                        "attachmentsDir": polylogue.get("attachmentsDir") or (str(attachments_dir) if expected_local_paths else None),
+                        "htmlPath": polylogue.get("htmlPath"),
+                        "contentHash": state_entry.get("contentHash") or content_hash,
+                        "localHash": existing.body_hash,
+                        "dirty": True,
+                    }
+                )
+                polylogue = _prune_empty_values(polylogue)
+                dirty_meta["polylogue"] = polylogue
```

Ruff (0.15.9)

[error] Undefined name `dirty_meta`

(F821)